Configuration/PyReleaseValidation/python/relval_2026.py

```diff
@@ -42,6 +42,8 @@
 #CloseByPGun for HGCAL
 numWFIB.extend([24896.0]) #CE_E_Front_120um D98
 numWFIB.extend([24900.0]) #CE_H_Coarse_Scint D98
+# NuGun 
+numWFIB.extend([20061.0]) #Nu Gun 2026D88
```

Comment:
Any specific reason to keep this workflow in D88, not in baseline?
(or because this PR has been done for quite a while?)

Reply:
Or you want to keep it in sync with patatrack which currently on D88 only? By the way, I think we move patatrack to all Phase-2 geometry already https://github.com/cms-sw/cmssw/pull/41011, maybe we can move patatrack baseline workflow to be the same baseline as other. 

If this is agree, may we can keep it here now, and I will make the move with patatrack baseline in the next PR.